packages/theme-language-server-common/src/hover/providers/RenderSnippetHoverProvider.ts

```diff
@@ -61,7 +61,7 @@ export class RenderSnippetHoverProvider implements BaseHoverProvider {
 
     if (liquidDoc.examples?.length) {
       const examples = liquidDoc.examples
-        ?.map(({ content }) => `\`\`\`liquid${content}\`\`\``)
+        ?.map(({ content }) => `\`\`\`liquid\n${content}\n\`\`\``)
```

Comment:
This was coming for 'free', but was more as a side effect of:
1) forcing examples to be on their own line.
2) including leading whitespaces in `content.value`

This made it really hard to have consistent behavior between descriptions and examples.